Tasks/Gradle/task.json

```diff
@@ -117,8 +117,8 @@
             "defaultValue": "build/classes/main/",
             "required": true,
             "groupName": "codeCoverage",
-            "helpMarkDown": "Comma separated list of directories containing class files, archive files(jar, war etc.). Directories and archives are searched recursively for class files. Code coverage is reported for class files present in the directories. For example, classes1/Tests,classes2/Tests2. Refer https://docs.gradle.org/current/userguide/jacoco_plugin.html.",
-            "visibleRule": "codeCoverageTool != None"
+            "helpMarkDown": "Comma separated list of directories containing class files, archive files(jar, war etc.). Directories and archives are searched recursively for class files. Code coverage is reported for class files present in the directories. By default it will be recursively searching for classes under build/main/classes which is default class directory for gradle builds",
```

Comment:
"Comma separated list of directories containing class files, archive files(jar, war etc.). Directories and archives are searched recursively for class files. Code coverage is reported for class files present in the directories. The task searches for classes under build/main/classes which is default class directory for gradle builds"